src/App.tsx

```diff
@@ -1,12 +1,19 @@
 import { Route, Routes } from "react-router-dom";
 import Home from "./pages/Home/Home";
 import "./styles/main.scss";
+import Nav from "./components/Nav/Nav";
 
 const App = () => {
   return (
-    <Routes>
-      <Route path="*" element={<Home />} />
-    </Routes>
+    <>
+      <Routes>
+        <Route path="/" element={<Home />} />
```

Comment:
This is not going to be the main route / is going to be for the login page create another route for home for now